sdk/search/search-documents/README.md

```diff
@@ -82,8 +77,50 @@ const indexClient = new SearchIndexClient(
 const serviceClient = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
 ```
 
+### Send your first search query
+To get running immediately, we're going to connect to a well known sandbox Search service provided by Microsoft.  This means you do not need an Azure subscription or Azure Cognitive Search service to try out this query.
+
+```js
+const { SearchIndexClient, AzureKeyCredential } = require("@azure/search-documents");
+
+// We'll connect to the Azure Cognitive Search public sandbox and send a
+// query to its "nycjobs" index built from a public dataset of available jobs
+// in New York.
+const serviceName = "azs-playground";
```

Comment:
minor: I think it's easier to just put the whole endpoint into a variable, since the portal gives them it as a URL rather than as a service name

Reply:
Done